assets/css/input.css

```
@@ -3,3 +3,47 @@
 @import 'flowbite/src/themes/default';
 @plugin "flowbite/plugin";
 @source "../../node_modules/flowbite";
+
+/* Highlight.js theme for code blocks */
+@import "highlight.js/styles/github-dark.css";
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)